### PR TITLE
[TF 2.0 API docs] expanding the tf.keras.activations.relu and tf.keras.activations.softmax documentation 

### PR DESCRIPTION
**Pull request by BlackHoundNate**
This addresses issue #26211 (https://github.com/tensorflow/tensorflow/issues/26211). I would be very happy to revise anything that's still incorrect here.

### CODE REVIEW

**Comment by BlackHoundNate** [comment]
I have now signed the CLA with both this account and the one from which I made the changes. Hopefully this comment triggers the CLA check to run again.

**Comment by NateMeyvis** [comment]
Re: "We need to confirm that all authors are ok with their commits being contributed to this project. Please have them confirm that here in the pull request."

Yes, I confirm that I am OK with my commits being contributed to this project.

**Comment by rthadur** [comment]
> I have now signed the CLA with both this account and the one from which I made the changes. Hopefully this comment triggers the CLA check to run again.

@BlackHoundNate did you modify using other username, i see there is a commit using @NateMeyvis , if that the case you need to sign the CLA for other usernames

**Comment by NateMeyvis** [comment]
> > I have now signed the CLA with both this account and the one from which I made the changes. Hopefully this comment triggers the CLA check to run again.
> 
> @BlackHoundNate did you modify using other username, i see there is a commit using @NateMeyvis , if that the case you need to sign the CLA for other usernames

I did modify using @NateMeyvis. Both of these accounts have signed the CLA (confirmed by an email I received at 10:08a). It appears that a project maintainer will have to change the CLA label to "yes;" is that correct? Again, if there's anything else i need to do, please just tell me.

**Comment by dynamicwebpaige** [comment]
Thank you for contributing, Nate! We appreciate it. 🙂

**Comment by BlackHoundNate** [comment]
> Thank you for contributing, Nate! We appreciate it. 🙂

Happy to help! I'm also eager to correct any mistakes I might have made. TensorFlow has been good to me and I'd like to give back a bit. Thanks for the note.

**Comment by BlackHoundNate** [comment]
First, I apologize in advance if the softmax docstring should have been a separate PR.

Second, the same CLA warning has just been generated because the PR still contains commits from both BlackHoundNate and NateMeyvis (sorry again). I have signed CLAs for both these accounts.

If there is anything else I should be doing, please let me know!

File: tensorflow/python/keras/activations.py [hunk 1]
```diff
@@ -42,17 +42,26 @@
 
 @keras_export('keras.activations.softmax')
 def softmax(x, axis=-1):
-  """Softmax activation function.
+  """Applies the softmax activation function.
+
+  This activation, often used to generate outputs representing probabilities, first applies the standard exponential function to the input and then normalizes the result.
```
Comment (tanzhenyu) [inline tensorflow/python/keras/activations.py]: Split this line please.

File: tensorflow/python/keras/activations.py [hunk 2]
```diff
@@ -42,17 +42,26 @@
...
+
+  ```python
+  t = tf.constant([[-1.1, 1.1, .7])
+  softmax(t).numpy() # ==> [[0.06220971, 0.5614435 , 0.37634683]]
```
Comment (tanzhenyu) [inline tensorflow/python/keras/activations.py]: This is showing up in 1.14 doc as well, correct?

File: tensorflow/python/keras/activations.py [hunk 3]
```diff
@@ -42,17 +42,26 @@
...
+  sum(sum(softmax(t))).numpy() # ==> 1.0
+  ```
 
   Arguments:
-      x : Input tensor.
-      axis: Integer, axis along which the softmax normalization is applied.
+      `x` : A `Tensor` or `Variable`.
```
Comment (tanzhenyu) [inline tensorflow/python/keras/activations.py]: I don't think you have to add single quote here.

File: tensorflow/python/keras/activations.py [hunk 4]
```diff
@@ -42,17 +42,26 @@
...
+      `axis`: An `integer` giving the axis along which the softmax normalization is to be applied.
```
Comment (tanzhenyu) [inline tensorflow/python/keras/activations.py]: ditto above.

File: tensorflow/python/keras/activations.py [hunk 5]
```diff
@@ -42,17 +42,26 @@
...
 
   Returns:
-      Tensor, output of softmax transformation.
+      A `Tensor` representing the softmax transformation of the input tensor.
```
Comment (tanzhenyu) [inline tensorflow/python/keras/activations.py]: ditto.

File: tensorflow/python/keras/activations.py [hunk 6]
```diff
@@ -141,24 +150,33 @@ def softsign(x):
 
 
 @keras_export('keras.activations.relu')
-def relu(x, alpha=0., max_value=None, threshold=0):
-  """Rectified Linear Unit.
+def relu(x, alpha=0., max_value=None, threshold=0.):
+  """Applies the ReLU activation function.
+
+  With default values, this returns the standard ReLU activation: the element-wise maximum of 0 and the input tensor. This activation is used in many graph architectures.
```
Comment (tanzhenyu) [inline tensorflow/python/keras/activations.py]: Split this line -- no more than 80 characters.

File: tensorflow/python/keras/activations.py [hunk 7]
```diff
@@ -141,24 +150,33 @@ def softsign(x):
...
 
-  With default values, it returns element-wise `max(x, 0)`.
+  Modifying default parameter values allows you to use thresholds other than zero, to set maximum values of the activation, and to use non-zero multiples of the input for values below the threshold.
```
Comment (tanzhenyu) [inline tensorflow/python/keras/activations.py]: ditto above.

**Comment by rthadur** [comment]
@BlackHoundNate could you please review comments , and resolve conflicts.

**Comment by rthadur** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!
